lib/ood_core/job/adapters/kubernetes/helper.rb

```diff
@@ -247,11 +249,20 @@ def submission_time(json_data)
 
   def pod_status_from_json(json_data)
     phase = json_data.dig(:status, :phase)
+    conditions = json_data.dig(:status, :conditions)
+    unschedulable = false
+    unless conditions.nil?
+      unschedulable = conditions.any? { |c| c.dig(:reason) == "Unschedulable" }
+    end
```

Comment:
I think this whole block can be `conditions.to_a.any? { |c| c.dig(:reason) == "Unschedulable" }` if we just cast to an array.

Reply:
I didn't realize `nil.to_a` resulted in an empty array.  That sounds newish but maybe it's always been that way.  I made the necessary adjustments.